src/DatePicker/Calendar.js

```diff
@@ -297,10 +289,12 @@ class Calendar extends Component {
       },
     };
 
-    const weekTitleDayStyle = prepareStyles(styles.weekTitleDay);
-    const weekTitleStyle = prepareStyles(styles.weekTitle);
-    const calendarContainerStyle = prepareStyles(styles.calendarContainer);
-    const yearContainerStyle = prepareStyles(styles.yearContainer);
+    prepareStyles(styles.root);
```

Comment:
Yeah, this function is supposed to be pure in production. That would be better to move those down into the JSX :+1:.